### PR TITLE
MULE-8545 Update spring schemas to 3.7

### PR DESCRIPTION
**Pull request by vromero**

### CODE REVIEW

File: modules/spring-config/src/main/resources/META-INF/spring.schemas [hunk 1]
```diff
@@ -9,11 +9,14 @@ http\://www.mulesoft.org/schema/mule/schemadoc/3.4/mule-schemadoc.xsd=META-INF/m
 http\://www.mulesoft.org/schema/mule/core/3.5/mule.xsd=META-INF/mule.xsd
 http\://www.mulesoft.org/schema/mule/schemadoc/3.5/mule-schemadoc.xsd=META-INF/mule-schemadoc.xsd
 http\://www.mulesoft.org/schema/mule/schemadoc/3.6/mule-schemadoc.xsd=META-INF/mule-schemadoc.xsd
+http\://www.mulesoft.org/schema/mule/core/3.7/mule.xsd=META-INF/mule.xsd
+http\://www.mulesoft.org/schema/mule/schemadoc/3.7/mule-schemadoc.xsd=META-INF/mule-schemadoc.xsd
 http\://www.mulesoft.org/schema/mule/core/current/mule.xsd=META-INF/mule.xsd
 http\://www.mulesoft.org/schema/mule/schemadoc/current/mule-schemadoc.xsd=META-INF/mule-schemadoc.xsd          ]
```
Comment (marianogonzalez) [inline modules/spring-config/src/main/resources/META-INF/spring.schemas]: what's the bracket in this line? was that already there?

File: patterns/core/src/main/resources/META-INF/spring.schemas [hunk 2]
```diff
@@ -4,4 +4,6 @@ http\://www.mulesoft.org/schema/mule/pattern/3.3/mule-pattern.xsd=META-INF/mule-
 http\://www.mulesoft.org/schema/mule/pattern/3.4/mule-pattern.xsd=META-INF/mule-pattern.xsd
 http\://www.mulesoft.org/schema/mule/pattern/3.5/mule-pattern.xsd=META-INF/mule-pattern.xsd
 http\://www.mulesoft.org/schema/mule/pattern/3.6/mule-pattern.xsd=META-INF/mule-pattern.xsd
+http\://www.mulesoft.org/schema/mule/pattern/3.7/mule-pattern.xsd=META-INF/mule-pattern.xsd
 http\://www.mulesoft.org/schema/mule/pattern/current/mule-pattern.xsd=META-INF/mule-pattern.xsd
+
```
Comment (marianogonzalez) [inline patterns/core/src/main/resources/META-INF/spring.schemas]: remove empty line

**Comment by marianogonzalez** [comment]
probably unrelated to this change, but there're schemas which imports other schemas (e.g: mule-json.xsd). We should upgrade the references there too

File: modules/spring-config/src/main/resources/META-INF/spring.schemas (hunk 1 above)
Comment (vromero) [inline modules/spring-config/src/main/resources/META-INF/spring.schemas]: good catch
it was already there but as per diff. I've updated: https://www.mulesoft.org/jira/browse/MULE-8544 to cover it and be able to backport

**Pull request merged**